README.md

```diff
@@ -85,3 +90,14 @@ Discuss features and ask questions on
 ## License
 
 MIT
+
+## Module Long Term Support Policy
+This module adopts the [
+Module Long Term Support (LTS)](http://github.com/CloudNativeJS/ModuleLTS) policy,
```

Comment:
when using the "view", the table doesn't look quite right.

Reply:
@dhmlau This is the preview I saw:

![screen shot 2018-11-16 at 3 12 39 pm](https://user-images.githubusercontent.com/12554153/48644929-19236280-e9b2-11e8-9215-8e8c2ecc19eb.png)